grails-app/conf/application.yml

```diff
@@ -16,8 +16,8 @@ environments:
     development:
         dataSource:
             dbCreate: update
-            driverClassName:  'com.mysql.jdbc.Driver'
-            url: jdbc:mysql://localhost/streama
+            driverClassName:  'com.mysql.cj.jdbc.Driver'
```

Comment:
@DrEthan77 Why have you changed the driver class? Is that a typo?

Reply:
![screen shot 2018-09-29 at 5 51 07 pm](https://user-images.githubusercontent.com/20546072/46251446-49b63980-c410-11e8-8348-7b214879b8ca.png)
nope, the old driver is depreciated

Reply:
Okay cool.